src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/Panel.tsx

```diff
@@ -22,13 +22,19 @@ import { Nag } from "ui/hooks/users";
 import { MAX_POINTS_FOR_FULL_ANALYSIS } from "protocol/thread/analysis";
 import { getFocusRegion } from "ui/reducers/timeline";
 import { ReplayClientContext } from "shared/client/ReplayClientContext";
+import { useStringPref } from "ui/hooks/settings";
 import { UnsafeFocusRegion } from "ui/state/timeline";
 import { getHitPointsForLocation } from "bvaughn-architecture-demo/src/suspense/PointsCache";
 
-const gutterOffset = 37;
+const lineNumberGutterAndBorderWidth = 31;
```

Comment:
This looks pretty sketchy.

Reply:
Yeah, I was more focused on an immediate fix 😇 these values should ideally come from system variables, but since it's not being used anywhere else I wanted to keep it local and simple for now.